### ISSUE TITLE
[Bug] Notice: Undefined variable: project in "Project management" section

### ISSUE THREAD

**Issue by orlov0562** [kanboard/kanboard#2623]
### Expected behaviour

No notices on the page

### Actual behaviour

Notices about undefined variable

### Steps to reproduce

1. Go to "Project management" section
2. Pay attention to the notices:
```
Notice: Undefined variable: project in /app/Template/project_list/show.php on line 4
Notice: Undefined variable: project in /app/Template/project_list/show.php on line 11
```

### Configuration

Copy and paste the configuration section from the Kanboard settings page or fill these fields:

- Application version: master
- PHP version: 7.0.8
- PHP SAPI: apache2handler
- OS version: Linux 3.10.0-327.22.2.el7.x86_64
- Database driver: sqlite
- Database version: 3.7.17
- Browser: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:48.0) Gecko/20100101 Firefox/48.0

**Comment by stinnux** [comment]
The data array has been incorrectly added to the hook. See PR #2625

**Comment by fguillot** [comment]
The fix is now available in Kanboard v1.0.33
